04-understanding-survey-data-documentation.Rmd

```diff
@@ -246,18 +235,19 @@ summary(anes_des)
 
 The next section of the survey documentation is the codebook. As mentioned above, the codebook provides information about the variables in a survey dataset. We can use it to select the variables for our analysis later on.
 
-## Searching for public-use survey data
+## Searching for public-use survey data 
```

Comment:
I'm not really a fan of this section.  I feel like there are two things conflated here: the data we use in the book, and where people can find data.

If we want to go where people find data then let's structure this differently such as:
- Many governments have publicly available data.  For example, in the U.S. you can get data directly from the various statistical agencies such as the Census Bureau or the CDC.  Or in the UK you can get data directly from the Office for National Statistics.
- There are also data repositories like ICPSR, Odum, others???
- Other websites have compiled a list of free data such as asdfree, others??